catkin_ws/src/stability/src/stability_node.cpp

```diff
@@ -0,0 +1,59 @@
+#include <ros/ros.h>
+#include <geometry_msgs/Wrench.h>
+#include <sensor_msgs/Imu.h>
+#include <std_msgs/Float64.h>
+#include <cmath>
+
+#include "MiniPID.h"
+#include "AxialPid.h"
+#include "DepthPid.h"
+
+int main(int argc, char **argv)
+{
+    RotPIDController rotCtrl;
+    rotCtrl.setPitchPID(0.1,0.0001,5);
+    rotCtrl.setRollPID(0.1,0.0001,5);
+    rotCtrl.setYawPID(0.1,0.0001,5);
+
+  std::cout<<"b"<<std::endl;
+
+  ros::init(argc, argv, "stability");
+
+  ros::NodeHandle n;
+
+  ros::Publisher motorPub = n.advertise<geometry_msgs::Wrench>("motorStability", 100);
+  //ros::Subscriber target = n.subscribe("target_topic", 50, &RotPIDController::setTargets, &rotCtrl);
```

Comment:
Should this still be commented out?

Reply:
that one will be useful for when we have nodes publishing where the robot should point, target_topic is just a filler so i thought i'd just keep it commented until we need it

Reply:
We should at least have some way to control the yaw goal via other nodes when we merge this. That way other nodes can be made that will work on that. I would suggest changing the name to something like 'orientation_target', or something that makes clear what target is being set (as opposed to depth target, position target, etc)

Reply:
ok, i changed the topic name